Sources/core/objects/OnMapBase.cpp

```diff
@@ -13,6 +13,22 @@ IdPtr<ITurf> IOnMapBase::GetTurf()
     return 0;
 }
 
+IdPtr<CubeTile> IOnMapBase::GetTile()
+{
+    IdPtr<IOnMapBase> item = GetId();
+    if (item.IsValid())
+    {
+        if (!item->GetOwner().IsValid())
+        {
+            return item;
+        }
+        else
+        {
+            return item->GetOwner()->GetTile();
+        }
```

Comment:
You can remove `else` here. Instead of `return item;` you can use `return GetId();`